HttpTrigger1/main.py

```diff
@@ -6,13 +6,21 @@
 
 
 from .models import Label, Participant, Meeting, create_tables, database_location
+from .auth import *
 
 ACCESS_TOKEN = os.environ["GITHUB_ACCESS_TOKEN"]
 REPO_FULL_NAME = "ElGarash/Jitsi-Meetings-Dashboard"
 TARGET_BRANCH = "gh-pages"
 
 
-def main(req: func.HttpRequest, context: func.Context) -> func.HttpResponse:
+def main(req: func.HttpRequest) -> func.HttpResponse:
+    try:
+        token = get_token_auth_header(req)
```

Comment:
This will need modification if you return the error instead of raising it.
```suggestion
       token_err, token = get_token_auth_header(req)
       if(token_err):
           return func.HttpResponse(token_err.message, status_code=token_err.status_code)
```

Other functions which can raise exceptions will need modification too.

Reply:
What is the problem with exception handling as a control flow mechanism?

Reply:
https://github.com/ElGarash/Jitsi-Meetings-Dashboard/pull/8#discussion_r676019573